packages/server-admin-ui/src/styles/_bootstrap-variables.scss

```
@@ -61,11 +61,196 @@ $theme-colors: (
+
+$sk-footer-color-dark: #dadbdd;
+$sk-footer-bg-dark: $sk-bg-dark;
+
+$sk-sidebar-color-dark: #e3e3e4;
+$sk-sidebar-bg-dark: $sk-bg-dark;
+$sk-sidebar-active-bg-dark: #2b303c;
+$sk-sidebar-border-color-dark: $sk-border-color-dark;
+
+$sk-progress-bg-dark: $gray-800;
+
+$sk-input-bg-dark: $sk-bg-dark;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)